### PR TITLE
Working branch

### PR DESCRIPTION
**Pull request by 410AngelaVu**

### CODE REVIEW

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## Hi @410AngelaVu  :wave: 

You've done well so far :+1:  and I commend you for that :clap: 
However, there are still some changes to be made before this project can be approved.

## REQUIRED

* [x] In case of next time, your commit messages should always be in **present** form, not **past**. For example, the first commits `Added User model and necessary gems', instead of `Add User model and necessary gems`.
* [x] Navigation links are present at the top of the page ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/navigation_links_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))
* [x] Your event model should have creation date as attribute ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/event_date_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))
* [x] In your migration, your events table should have creator_id as the foreign key ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_assoc_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))

##### UsersController

###### `show` action

* [x] lists events created by user, i.e. `@created_events = current_user.events` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/current_user_show_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))
* [x] lists events previoulsy attendented by user **using event model scope**, i.e. `@past_events = current_user.attended_events.previous` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_events_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))
* [x] lists upcoming events previously attended by user **using event model scope**, i.e. `@upcoming_events = current_user.attended_events.upcoming` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_events_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))


### Make the required changes and submit for another review.
## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

**Review by ebeagusamuel — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
### Hey there :wave:
#### There is a bug in your code that is causing it to break when a user tries to create a new event. It is breaking with an "SQLite3::SQLException: no such table: main.creators". Please check your code and fix this issue.
#### Do request for another review when you are done.
slack(@ebeagusamuel)

**Review by LamiaSristy — CHANGES_REQUESTED** [review]
# Changes Required 🔴 

Hello @410AngelaVu.
As my previous TSE mentioned, there is a bug in your code that is causing it to break when a user tries to create a new event. 
It is breaking with an "SQLite3::SQLException: no such table: main.creators". 
Please check your code and fix this issue.
Please make the changes and then again ask for a new review.
If you have any queries, feel free to ask @lamiasristy
Happy Coding 👍

File: app/views/home/index.html.erb
Comment (teekaytech) [inline app/views/home/index.html.erb]: Use the helper module for all view-related logic.

File: app/controllers/events_controller.rb
Comment (teekaytech) [inline app/controllers/events_controller.rb]: Refactor the @future_events and @previous_events to use your event scopes:
- lists past events, i.e. `@previous_events = Event.previous`
-  lists ucomping events, i.e. `@future_events = Event.upcoming`

File: app/controllers/users_controller.rb
Comment (teekaytech) [inline app/controllers/users_controller.rb]: Move the `@upcoming_events` and `@past_events` instance variables from the index to show action since it is the user's show page that displays events the user has created and events that the user is attending to separated in past and upcoming

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__So far, you've done really well in this project.
However, some changes are required before it can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] The `Sign In` button should be renamed to `Sign Up`



- [x] The `Schedule new session` button should be renamed to `Create New Event`




- [x] Navigation links/bar is not present at the top of some of the pages (e.g. http://localhost:3000/events). Make the navbar present on all pages.
- [x]  The user's show page displays **events the user has created** and **events that the user is attending to separated in past and upcoming**

- [x] Check the inline comments below for other changes.


* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by cristianCeamatu — APPROVED** [review]
### 🥇 APPROVED 🥇 Congratulations and good luck with the next projects!

If you ever need any help or notice that you are wasting too many hours on a bug don't hesitate to contact me on Slack.

Best regards,
Cristian Ceamatu
Slack: Cristian Viorel Ceamatu

**Pull request merged**
